plugins/katello/3.14/installation/index.md

```diff
@@ -170,6 +170,68 @@ arguments. Multiple entries can be entered via
+
+When the foreman-installer is run, it is deployed with a `default` predefined tuning profile. Other than the default tuned profile, foreman-installer supports 4 different tuning profiles:
+
+  * medium
+  * large
+  * extra-large
+  * extra-extra-large
+
+Based on your environment needs, use one of the tuning profiles (`medium`, `large`, `extra-large`, `extra-extra-large`) in the installer.  For example, `medium` profile can be applied like:
+
+```bash
+foreman-installer --tuning medium
+```
+
+To reset to the default profile:
+
+```bash
+foreman-installer --tuning default
+```
+
+Use `foreman-installer --help | grep tuning` to identify the current tuning level.
+
+Sample output for `medium` tuning:
+
+```bash
+foreman-installer --help | grep tuning
+    --tuning INSTALLATION_SIZE    Tune for an installation size. Choices: default, medium, large, extra-large, extra-extra-large (default: "medium")
+```
+
+Sample output for `default` tuning:
+
+```bash
+foreman-installer --help | grep tuning
+    --tuning INSTALLATION_SIZE    Tune for an installation size. Choices: default, medium, large, extra-large, extra-extra-large (default: "default")
+```
+
+**Note**
+
+- Definitions of various tuning profiles can be found in this directory `/usr/share/foreman-installer/config/foreman.hiera/tuning/sizes/`.  Note that `common.yaml` is always applied and the selected tuning profile (e.g., `medium`) is applied on top and takes precedence.
+- Using the `--tuning` option does not update `/etc/foreman-installer/custom-hiera.yml`, instead it directly updates the required configuration as specified in the corresponding tuning profile. You can still use `custom-hiera.yml` to override any configuration if really needed.
+- If you had already used `custom-hiera.yml` and starting to use the tuned profiles, you may want to review the definition of tuned profiles (`/usr/share/foreman-installer/config/foreman.hiera/tuning/common.yaml` and `/usr/share/foreman-installer/config/foreman.hiera/tuning/sizes/`) and remove the duplicated configuration entries from your `custom-hiera.yml`.
+- You can also optionally use `foreman-installer --noop` to run the installer in a test mode and identify what configurations will be changed before actually running the installer.
+
+### Which tuning profile should you choose?
+
+It is difficult to find the exact tuning profile for a specific environment in the first attempt because it depends on various factors like the number of managed hosts, the features used in scale (E.g., Remote Execution), the bulk action on hosts, the total amount of content, amount of host traffic to foreman, etc.  Our recommendation is that you start with the tuning profile guidance as shown in the below table based on the number of managed hosts and scale up your environment as needed.
```

Comment:
`the bulk action` looks odd. Did you mean `bulk actions`?
```suggestion
It is difficult to find the exact tuning profile for a specific environment in the first attempt because it depends on various factors like the number of managed hosts, the features used in scale (E.g., Remote Execution), bulk actions on hosts, the total amount of content, amount of host traffic to foreman, etc.  Our recommendation is that you start with the tuning profile guidance as shown in the below table based on the number of managed hosts and scale up your environment as needed.
```

Reply:
done